score/audit-checklist.md

```diff
@@ -476,6 +477,22 @@ self.something.set( str(Something()) )
 self.something.set( Something() )
 ```
 
+The second is when you store an unordered collection of items in StateDB, such as a python set datatype. Generally, set datatype is converted to list datatype and serialized using json.dumps and saved in StateDB. However, the result of converting from set to list may not be the same for each node. Therefore, the data must be manipulated so that the same result can be stored in StateDB.
+
+```python
+self.test_var = VarDB('test_var', db, str)
```

Comment:
I think it would be better to replace `self.test_var = VarDB('test_var', db, str)` with `self.test_var = VarDB('test_var', db, value_type=str)` for unification with [icondev.io (token-crowdsale guide)](https://www.icondev.io/docs/token-crowdsale).